src/pdf.js

```diff
@@ -13,11 +13,12 @@
  * limitations under the License.
  */
 /*jshint globalstrict: false */
```

Comment:
Can we add a space before `jshint` here too to be consistent with the rest of the codebase?